vru_use_case_scripts/plot_time_to_collision

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+from dataclasses import dataclass
+import itertools
+import math
+from pathlib import Path
+
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.interpolate import splrep, BSpline
+
+# This can be used to establish what constitutes a collision
+COLLISION_THRESHOLD_M = 5.0
```

Comment:
I don't think the parameter data is recorded in any of the topics. But the length of vehicle is 5 meters
https://github.com/usdot-fhwa-stol/carma-config/blob/develop/ail_vru_uc1_scenario/VehicleConfigParams.yaml#L22
Then the radius should be 5 / 2 + 1.2 (travel dist for pedestrian for 1s taken as its diameter) / 2 = 3.1